### ISSUE TITLE
Fork/rename `rubocop-ast` gem

### ISSUE THREAD

**Issue by mikowitz** [ruby-lint/ruby_lint#14]
RubyLint depends on `rubocop-hq/rubocop-ast`. We should fork this gem as well and perform the same renaming on it as well.

**Comment by listrophy** [comment]
I’d personally put this off until later, in order to keep down scope. Definitely worth (eventually)  addressing though!

**Comment by mikowitz** [comment]
Yeah, absolutely increases scope quite a bit. Came across the issue while doing some of the renaming work and wanted to at least note it for the future.

**Comment by veganstraightedge** [comment]
@mikowitz I agree that if this renaming effort succeeds, it should go all the way. And @listrophy I agree with you about putting it off for later.

Thanks for creating the issue as a reminder for the future. 👍🏻
